saas/controllers/main.py

```diff
@@ -21,16 +22,26 @@ def auth_to_build(self, build_id=None):
 
         return env.get_template('saas_auth.html').render({'build_url': build_url})
 
-    @route('/saas/create-fast-build/<int:template_operator_id>', type='http', auth='public')
-    def create_fast_build(self, template_operator_id, **kwargs):
+    @route(['/saas/<model("saas.template"):template_id>/create-fast-build',
+            '/saas/<model("saas.template"):template_id>/<model("saas.template.operator"):template_operator_id>'
+            '/create-fast-build'], type='http', auth='public')
```

Comment:
Make template_operator_id optional (pass as argument, not as a path). How we can skip it otherwise? ``/saas/1//create-fast-build``?

Reply:
so we have several routes `/saas/<model("saas.template"):template_id>/create-fast-build` and `/saas/<model("saas.template"):template_id>/<model("saas.template.operator"):template_operator_id>/create-fast-build`. If we don't have template_operator_id, so we can write just `saas/1/create-fast-build`

Reply:
I made several routes so we can use this: <model("saas.template.operator"):template_operator_id>

Reply:
template_operator_id is None by default

Reply:
Ok, I see.
By the way, where we can get id for template_operator_id?

Reply:
https://github.com/it-projects-llc/saas-addons/blob/e09689c26d4ec0bd28bf5851cec0b4da931e6599/saas/controllers/main.py#L30